apps/src/code-studio/videos.js

```diff
@@ -5,7 +5,7 @@ import trackEvent from '../util/trackEvent';
 import React from 'react';
 import ReactDOM from 'react-dom';
 import FallbackPlayerCaptionDialogLink from '../templates/FallbackPlayerCaptionDialogLink';
-var videojs = require('video.js');
+var videojs = require('video.js').default;
```

Comment:
The need for `.default` here suggests we might be able to switch to `import` syntax here.  Not 100% sure though.

```suggestion
import videojs from 'video.js';
```

Reply:
Good call.